### DISCUSSION TITLE
Lively API values into Unity/Godot?

### DISCUSSION THREAD

**Discussion by Symmasolan** [rocksdanister/lively#1766] [Q&A] [answered]
I know that the wiki guide is literally named "Web Guide", but the way the README is worded for the [Lively API](https://github.com/rocksdanister/lively#lively-api) feature section gives me some small hope that it's possible to get the API System and/or Interaction Data values in a Unity-driven wallpaper.

Is it at all possible to get the Interaction or System Data values from the Lively API in a Unity (or Godot) project?

**Comment by rocksdanister** [comment]
Its possible to use simple ipc like namedpipes to do this (infact this is what mpv video player is already using).. not coded yet since no one asked.

**Reply by Symmasolan** [reply]
Is it possible this could be added as a feature suggestion for the future? I'd love to make a few Unity-based wallpapers using the API. I understand if not though, since this (awesome!) project has been around for a decent while and I seem to have been the first person to ask for it.

**Reply by rocksdanister** [reply, marked as answer]
Moved here: https://github.com/rocksdanister/lively/issues/1767
